core/custom_test.go

```diff
@@ -353,3 +357,159 @@ func TestErrorBadMapFormat(t *testing.T) {
 	_, err := CustomMapFromStrings(inputs)
 	then.Err(t, errInvalidCustomFormat, err)
 }
+
+func TestThemeScrollWithFewerThan10Choices(t *testing.T) {
+	choices := []choose.Choice{
+		{Text: "Option 1"},
+		{Text: "Option 2"},
+		{Text: "Option 3"},
+	}
+
+	result := ThemeScroll(choices, 0) // cursor at index 0
+
+	// Should show all 3 options with the second one selected
+	then.True(t, strings.Contains(result, "[•] Option 1"))
+	then.True(t, strings.Contains(result, "[ ] Option 2"))
+	then.True(t, strings.Contains(result, "[ ] Option 3"))
+	then.False(t, strings.Contains(result, "Option 4"))
+}
+
+func TestThemeScrollWithExactly10Choices(t *testing.T) {
+	choices := make([]choose.Choice, 10)
+	for i := 0; i < 10; i++ {
```

Comment:
```suggestion
	for i := range 10 {
```

Can you use the new 1.24 range syntax for for loops